test/test_images/grpc-ping/proto/ping.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package ping;
```

Comment:
Can you include info on the command you wrote to produce `ping.pb.go`?  Maybe even add a `// go:generate` comment (I think adding it to `./hack/update-codegen.sh` is probably overkill)

Reply:
Copied from https://github.com/knative/docs/tree/master/serving/samples/grpc-ping-go 😃  

I'll look up how to actually generate it and add the comment.

Reply:
Can we make sure this sample works with 0.4, and I'll link it from the release notes 🔥